pkgs/tools/misc/qflipper/default.nix

```diff
@@ -19,12 +19,14 @@
 , qtquickcontrols2
 , qtgraphicaleffects
 , qtwayland
+, nix-update-script
 }:
 let
-  version = "1.0.1";
+  pname = "qFlipper";
+  version = "1.0.2";
   timestamp = "99999999999";
   commit = "nix-${version}";
-  hash = "sha256-vHBlrtQ06kjjXXGL/jSdpAPHgqb7Vn1c6jXZVXwxHPQ=";
+  sha256 = "vHBlrtQ06kjjXXGL/jSdpAPHgqb7Vn1c6jXZVXwxHPQ=";
```

Comment:
Please revert that. sri hashes should have a prefix.

Reply:
@SuperSandro2000 updaterScript could not process SRI w/o hash — and I thought that if you specify hash type, you can waiver type from the hash itself  :\

Reply:
Oh wait, I guess it did not, the hash is the same.

Reply:
Ok, tautology did work, but it's not pretty :(